src/env.ts

```diff
@@ -0,0 +1,25 @@
+import { createEnv } from '@t3-oss/env-nextjs';
+import { z } from 'zod';
+
+export const env = createEnv({
+  server: {
+    DATABASE_URL: z.string().min(1),
```

Comment:
custom zod validations for each env e.g. min length for secret might be different , POSTGRES_URL should be a url as such

Reply:
cool